### ISSUE TITLE
/Users/wd/.ssh/config: line 201: Bad configuration option: s

### ISSUE THREAD

**Issue by wd** [moul/advanced-ssh-config#191]
Maybe this is a ssh limit, when there is a long comment, you will get this error.

`.ssh/config` file:
```
Host *.com
  # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs   fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df  sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp # sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp sdf sd fs fds df sdf sd fs dfs dfsdfsfyypp
```

And then
```
$ /usr/bin/ssh abc.com
/Users/wd/.ssh/config: line 8: Bad configuration option: d
/Users/wd/.ssh/config: terminating, 1 bad configuration options
```

When you have a lots of `KnownHosts` for a host, you will get this problem.

**Comment by wd** [comment]
To fix this, just rerun `assh config build > ~/.ssh/config`, but after you connect enough hosts, the problem will occur again.

**Comment by moul** [comment]
Hi @wd, thank you for your feedback,

Can you give me the related part of your `assh.yml` (or *included*) file(s) ?

**Comment by wd** [comment]
```
$ cat assh.yml
hosts:
  '*.com':
    IdentityFile: ~/.ssh/id_rsa
```

```
$ cat config
# This file was automatically generated by assh v2.5.0
# on 2017-01-10 17:16:51 +0800 CST, based on ~/.ssh/assh.yml
#
# more info: https://github.com/moul/advanced-ssh-config

# host-based configuration
Host *.com
  IdentityFile ~/.ssh/id_rsa
  # KnownHosts: [a.com]

Host a.com
  IdentityFile ~/.ssh/id_rsa
  # KnownHostOf: *.com

# global configuration
Host *
  ProxyCommand assh connect --port=%p %h
```

When you have lots of host ended with `*.com`, assh will add lots at line `# KnownHosts: [a.com]`.

**Comment by sh1ny** [comment]
Hello,

I can confirm the same, i have a lot of wildcards guessing like :

Host sc*-*comp*
  IdentityFile ~/.ssh/rsa
  Port 22
  StrictHostKeyChecking no
  User root
  # Inherits: [root-template]

We're talking several thousand hosts matched by this. Also this breaks when trying to use something like ansible on top, because i can't re-run assh build config all the time.

Is there any easy fix, or should we wait for a new release ?

**Comment by moul** [comment]
I just pushed a fix for this (#192); can you check that it fixed your problem?

**Comment by wd** [comment]
I've tested, it's fixed. I see you split `# KnownHosts` to several lines. Thanks.
